### DISCUSSION TITLE
Pulling Data a lot slower with DataSync

### DISCUSSION THREAD

**Discussion by Brian-Con** [Azure/azure-mobile-apps#690] [Q&A]
Hi,

I'm in the process of migrating from Microsoft.Azure.Mobile.Server to Microsoft.AspNetCore.DataSync. I have converted both my server-side code and client-side code. I'm finding that it is taking a lot longer to perform a Pull of the data with the DataSync version.

A table with roughly 2,000 records takes about 13 seconds to download with Azure.Mobile, but 43 seconds with DataSync.

The issue/delay 'seems' to be due to the DataSync update on the client-side because if I run the Azure.Mobile client against the DataSync server version it takes about 13 seconds.

Thanks,
Brian

**Comment by adrianhall** [comment]
This is being discussed in https://github.com/Azure/azure-mobile-apps/discussions/689
